### ISSUE TITLE
Bug on s3_bucket_id output

### ISSUE THREAD

**Issue by atrakic** [terraform-aws-modules/terraform-aws-s3-bucket#89]
## Description

Please provide a clear and concise description of the issue you are encountering, your current setup, and what steps led up to the issue. If you can provide a reproduction, that will help tremendously.

## ⚠️ Note

Before you submit an issue, please perform the following first:

1. Remove the local `.terraform` directory (! ONLY if state is stored remotely, which hopefully you are following that best practice!): `rm -rf .terraform/`
2. Re-initialize the project root to pull down modules: `terraform init`
3. Re-attempt your terraform plan or apply and check if the issue still persists

## Versions

 - Terraform:

 - Provider(s):

 - Module:

## Expected behavior

The name of the bucket is should be a string.

## Additional context

It should be:
```
output "s3_bucket_id" {
  description = "The name of the bucket."
  value       = element(aws_s3_bucket.this.*.id, [""]), 0)
}
```

**Comment by antonbabenko** [comment]
I am not sure what is the problem here but please fill in the template when opening an issue (not just copy-paste).

**Comment by atrakic** [comment]
Sure, let me try again then.
